lib/twitter_cldr/js/tasks/tasks.rb

```diff
@@ -51,28 +51,19 @@ def build(options = {})
           build_duration = time_operation do
             options[:files].each_pair do |file_pattern, minify|
               compiler.compile_each(:minify => minify) do |bundle, locale|
-                out_file = File.join(output_dir, file_pattern % locale)
-                FileUtils.mkdir_p(File.dirname(out_file))
-                File.open(out_file, "w+") do |f|
-                  f.write(bundle.source)
-                end
+                out_file_path = [output_dir, file_pattern % locale]
+                write_bundle(out_file_path, bundle.source)
```

Comment:
Agreed, refatoring to call `write_file` directly. No point having redundant filler methods.